Sources/Core/AWSClientRuntime/Auth/Credentials.swift

```diff
@@ -6,34 +6,35 @@
 //
 
 import Foundation
+import ClientRuntime
 
 public typealias AWSCredentials = Credentials
 
 /// A type representing credentials for authenticating with an AWS service
 /// 
 /// For more information see [AWS security credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html#AccessKeys)
-public struct Credentials {
+public struct Credentials: Identity {
     let accessKey: String
     let secret: String
-    let expirationTimeout: Date?
     let sessionToken: String?
+    public let expiration: ClientRuntime.Date?
```

Comment:
What is a `ClientRuntime.Date` and why can't we just use standard `Foundation.Date` for expiration?

Naming our own type the same as a Foundation type is a recipe for confusion... I'm guessing the compiler flagged it to you because `import Foundation` is at the top of this source file.